packages/compass-e2e-tests/helpers/compass.js

```diff
@@ -159,15 +163,18 @@ async function startCompass(
   const _stop = app.stop.bind(app);
 
   app.stop = async () => {
+    const logs = await app.client.getMainProcessLogs();
     if (shouldStoreAppLogs) {
       const logPath = path.join(LOG_PATH, `electron-main.${nowFormatted}.log`);
       debug(`Writing application main process log to ${logPath}`);
-      const logs = await app.client.getMainProcessLogs();
       await fs.writeFile(logPath, logs.join('\n'));
     }
     debug('Stopping Compass application');
     await _stop();
     debug('Removing user data');
+
+    app.compassLog = await getCompassLog(logs);
```

Comment:
Do you think it would be worth storing this log in the `if (shouldStoreAppLogs) { ...` block also for debugging purposes (similar to what we do with app logs)?

Reply:
Yes, absolutely! :+1: Added a commit for that.